modules/javafx.media/src/main/java/com/sun/media/jfxmediaimpl/HostUtils.java

```diff
@@ -57,19 +59,19 @@ public static boolean is64Bit() {
     }
 
     public static boolean isWindows() {
-        return osName.startsWith("windows");
```

Comment:
`osName` is unused now

Reply:
`osArch` is also unused. It's just a local variable.

Reply:
@nlisker osArch seems to be used for is64bit still

Reply:
Yes, but what @nlisker means is that `osArch` does not need to be a member variable but only a local one inside the static initializer above.